smithy-jsonschema/src/main/java/software/amazon/smithy/jsonschema/JsonSchemaConfig.java

```diff
@@ -368,7 +371,12 @@ public void setSupportNonNumericFloats(boolean supportNonNumericFloats) {
         this.supportNonNumericFloats = supportNonNumericFloats;
     }
 
+    @Deprecated
     public boolean isEnableOutOfServiceReferences() {
+        if (enableOutOfServiceReferences) {
+            LOGGER.warning("Deprecated JSON Schema setting `enableOutOfServiceReferences` enabled. "
+                    + "Bind shapes directly to the service instead.");
+        }
```

Comment:
Were there any docs for this that need updated?

Reply:
No, this was a code-only configuration option that was provided.